AEPTarget/Sources/TargetDeliveryRequestBuilder.swift

```diff
@@ -54,7 +55,17 @@ enum TargetDeliveryRequestBuilder {
             property = Property(token: propertyToken)
         }
 
-        return TargetDeliveryRequest(id: targetIDs, context: context, experienceCloud: experienceCloud, prefetch: prefetch, execute: execute, notifications: notifications, environmentId: environmentId, property: property)
+        var qaMode: [String: AnyCodable]?
+        if
+            let qaModeParameters = qaModeParameters,
+           !qaModeParameters.isEmpty,
+           let qaModeData = qaModeParameters.data(using: .utf8)
```

Comment:
this formatting is rough - is it from swift format?  wondering if we need to revisit our rules if so...

Reply:
Yeah, i've fixed the alignment here but it's essentially [opening_brace](https://realm.github.io/SwiftLint/opening_brace.html) (swiftlint) and [wrapMultilineStatementBraces](https://github.com/nicklockwood/SwiftFormat/blob/master/Rules.md#wrapmultilinestatementbraces) which are at odds here. I've contributed little to the formatting/ linting rules here and I don't think we have a standard (rules) that we follow across the extensions here.
I can essentially disable wrapMultilineStatementBraces in swiftformat config if it's desirable, let me know.